pilot/pkg/features/pilot.go

```diff
@@ -301,9 +301,11 @@ var (
 		"If enabled, pilot will set the incremental flag of the options in the mcp controller "+
 			"to true, and then galley may push data incrementally, it depends on whether the "+
 			"resource supports incremental. By default, this is false.").Get()
-
+	// CentralIstioD will be Deprecated: TODO remove in 1.9 in favor of `ExternalIstioD`
```

Comment:
if you have not already - please file a bug for this deprecation and mark it staleproof. Label it for environments, the 1.9 release, with release blocker as a marker and assign to yourself.

Reply:
I think in this pr you can mark it deprecated in the env desc

Reply:
yep it is deprecated with this PR, what @sdake meant is for removal in 1.9.